Update URL for Soitoras Dark Theme

Soitora opened the pull request:
Just updated the URL from GitCDN to GitHub Pages as per #25. 

I would like to exclude the theme from your backup server as it's the default option that isn't up-to-date (in fact: 7 months old version) – but I could not figure out how to only make your backup server only apply to Blargmodes Dark Theme.

SimonAlling commented:
I certainly understand that you want your changes to be reflected for BSC users. I'll make the backup server option disabled by default after merging this PR.

Soitora commented:
I am not experienced with Git so that was me trying to get up-to-date with your branch, which I realised was not the way to go about it, but I don't know how to "leave out" a commit either :s

SimonAlling commented:
Aha. Ingen fara. Alla har vi varit där – Git är rätt svårt så fort man går bortom `add`, `commit` och `status` typ. Ska se om jag kan hitta en bra lösning.

Soitora commented:
I think I succeded.

I rebased with `git rebase -i --rebase-merges HEAD~2`, I removed the two changes I made to the patch (somehow only removing the merge didn't work) and then republished the change to the URL. (It automatically closed this PR however)
